dev/scripts/setup_unleash.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+import os
+
+import psycopg2
+import requests
+from requests.adapters import HTTPAdapter
+from requests.adapters import Retry
+
+
+unleash_pat = os.getenv("UNLEASH_PAT") or "user:6188b62f2f59348f3c195b66983147111682f4bb78a3f7ed9626bd84"
+assert isinstance(unleash_pat, str)
+data = {
```

Comment:
Why make it more complicated than it needs to be?

Reply:
Why make it more complicated than it needs to be? I'm fine with leaving it defined as a dict here because this is the only thing that will use this data.